src/sonic-config-engine/tests/simple-sample-graph.xml

```diff
@@ -105,6 +132,11 @@
         <a:Hostname>ARISTA04T1</a:Hostname>
         <a:RouteMaps/>
       </a:BGPRouterDeclaration>
+      <a:BGPRouterDeclaration>
+        <a:ASN>65100</a:ASN>
+        <a:Hostname>CHASSIS_PEER</a:Hostname>
```

Comment:
This hostname name is of remote Linecard ? What will be the hostname for multi ASIC Linecard.

Reply:
I don't fully understand the question. The hostname would be what someone picked for it. Are you saying you want this name changed?